### DISCUSSION TITLE
First steps after Use template

### DISCUSSION THREAD

**Discussion by ricardofrantz** [colinwilson/lotusdocs#110] [Q&A]
Hi, I need to build a new documentation website and I decided to try Lotus Docs. I selected **Use this template** -> **Create a new repository** and then in **Settings** -> **Pages** -> **Deploy from a branch** and then **release / (root)**, which seems to work fine as the website was successfully deployed to https://nekstab.github.io/Doc/ 

Now, the question is what is the best approach to edit the website. Following the instructions in the README.md is not clear in these conditions. Could someone more experienced with this guide me ? It seems I do not need to follow with 'hugo new, mod init, etc ? How can I clear the contend and add the sections I need? 

Thanks for your help,
Ricardo

**Comment by Funasitien** [comment]
You need to select action and not bra ch in the pages settings
